### ISSUE TITLE
histogram plot for Preactivation/activation layer

### ISSUE THREAD

**Issue by neerajtiwari360** [tensorflow/tensorboard#3148]
Well after going through tensorboard with tensorflow2.0, I am unable to plot the activation histogram for any layer, I can only see the tab for Conv and batch normalization.
can you please support me for activation visualization, in tensorflow2.0.

**Comment by rmothukuru** [comment]
In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by rmothukuru** [comment]
@neerajtiwari360,
Can you please provide the information asked in the above comment. Thanks!

**Comment by rmothukuru** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
